selfdrive/car/subaru/values.py

```diff
@@ -6,6 +6,7 @@
 
 class CAR:
   IMPREZA = "SUBARU IMPREZA LIMITED 2019"
+  FORESTER_2019 = "SUBARU FORESTER 2019"
```

Comment:
The year is unnecessary here

Reply:
There is a preglobal PR (https://github.com/commaai/openpilot/pull/1489) which is already using FORESTER

Reply:
I think global should have priority given that it's the newest platform.

Reply:
I removed the model year